logicrunner/handle_calls.go

```diff
@@ -30,6 +30,11 @@ import (
 	"go.opencensus.io/trace"
 )
 
+// PSE AGN means "please again" in radio jargon.
+// This text was chosen because it's very unlikely anyone
+// will use the same text for anything else in our code base.
+var ErrRetry = errors.New("PSE AGN")
```

Comment:
why don't use common message? if we start to use any jargons that we likes we also can start write code/comments in different languages